app-editors/vim/vim-8.0.0046.recipe

```diff
@@ -0,0 +1,103 @@
+SUMMARY="A highly configurable text editor"
+DESCRIPTION="Vim is a highly configurable text editor built to enable \
+efficient text editing. It is an improved version of the vi editor \
+distributed with most UNIX systems.
+
+Vim is often called a \"programmer's editor,\" and so useful for programming \
+that many consider it an entire IDE. It's not just for programmers, though. \
+Vim is perfect for all kinds of text editing, from composing email to editing \
+configuration files.
+Despite this, Vim can be configured to work in a very simple (Notepad-like) \
+way, called evim or Easy Vim."
+HOMEPAGE="http://www.vim.org"
+COPYRIGHT="Bram Moleenar et al."
+LICENSE="Vim"
+REVISION="1"
+SOURCE_URI="https://github.com/vim/vim/archive/v8.0.0046.tar.gz"
+CHECKSUM_SHA256="a768d3b959c44c6084c1c745012766a537303c687f8869d3e877c5a25cc812f4"
+SOURCE_DIR="vim-8.0.0046"
+PATCHES="vim-8.0.0046.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	vim = $portVersion
+	cmd:ex = $portVersion
+	cmd:gview = $portVersion
+	cmd:gvim = $portVersion
+	cmd:gvimdiff = $portVersion
+	cmd:rgview = $portVersion
+	cmd:rgvim = $portVersion
+	cmd:rview = $portVersion
+	cmd:rvim = $portVersion
+	cmd:vi = $portVersion
+	cmd:view = $portVersion
+	cmd:vim = $portVersion
+	cmd:vimdiff = $portVersion
+	cmd:vimtutor = $portVersion
+	cmd:xxd = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
```

Comment:
@miqlas, the reason why we don't need **`$secondaryArchSuffix`** is because the following conditions are **_all_** met:
* the recipe builds on all primary architectures,
* and the recipe provides no **`lib:lib*`** nor **`devel:lib*`**.

So if you have successfully built the recipe on x86_gcc2 primary arch (and x86 primary arch) then you should proceed with the removal of all occurences of **`$secondaryArchSuffix`** (and **`${secondaryArchSuffix}`** of course.)

BTW, if a recipe does not have a non empty SECONDARY_ARCHITECTURES then **`$secondaryArchSuffix`**  is always empty. This is why we prefer to remove them when they are not useful.